owsi-core/owsi-core-components/owsi-core-component-commons/src/main/java/fr/openwide/core/commons/util/binding/BindingUtils.java

```diff
@@ -0,0 +1,20 @@
+package fr.openwide.core.commons.util.binding;
+
+import org.bindgen.Binding;
+
+public final class BindingUtils {
+
+	public static Class<?> getRootType(Binding<?> binding) {
```

Comment:
On a second thought, we could add this as an instance method to `AbstractCoreBinding`, which is a parent class to every binding. What do you think?